Dockerfile

```diff
@@ -20,8 +20,9 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 # STAGE 1 - install build dependencies and activate virtualenv
-ARG         ALPINE_IMAGE=artifactory.algol60.net/docker.io/library/alpine:3.17
```

Comment:
I think it fails if you do not pass a a build arg?  This is just meant to be the default but is overridden by our makefile.  I'm not even sure it should be the artifactory one as opposed to the publicly available image, but nonetheless I think it is good to keep a default

Reply:
When I added `ARG PYTHON_VERSION` to the `Dockerfile` it wasn't actually getting set, and `PYTHON_VERSION` and `ALPINE_IMAGE` were being passed exactly the same way.

I tried these variations:

```Dockerfile
ARG         PYTHON_VERSION
ARG         ALPINE_IMAGE=artifactory.algol60.net/docker.io/library/alpine:3.17
FROM       ALPINE_IMAGE
```


```Dockerfile

ARG         ALPINE_IMAGE=artifactory.algol60.net/docker.io/library/alpine:3.17
FROM        ALPINE_IMAGE
ARG         PYTHON_VERSION
```

Neither resulted in a working `PYTHON_VERSION` until I changed the quote situation in the `Makefile`. So I took that as `ALPINE_IMAGE` was not working either, and it was masking the issue by using a default string (e.g. it looked like it was working). I don't think it was getting overridden at all.


As for the image, we should use the Artifactory one. In general we should use our Artifactory images, so we aren't dependent on other hosts connectivity and maintenance. This is what casm-devops has been advocating for.

Reply:
Restored the default.